src/swarm-lang/Swarm/Language/Syntax/AST.hs

```diff
@@ -71,6 +71,8 @@ data Term' ty
     TAntiText Text
   | -- | A Boolean literal.
     TBool Bool
+  | -- | An antiquoted Haskell variable name of type Syntax.
+    TAntiSyn Text
```

Comment:
Why not `TAntiVarName`?

Reply:
All antiquotations are variable names; the interesting part is the *type* of the antiquoted variable.